ui/StatusQ/src/StatusQ/Components/StatusDraggableListItem.qml

```diff
@@ -180,9 +191,9 @@ ItemDelegate {
 
     background: Rectangle {
         color: "transparent"
-        border.width: 1
-        border.color: Theme.palette.baseColor2
-        radius: 8
+        border.width: customizable ? 0 : 1
```

Comment:
Just setting `border.width: 0` would hide it

Reply:
I don't get this? don't you need it for the social links delegate?

Reply:
Yes but you're also setting:
`border.color: customizable ? "" : Theme.palette.baseColor2`

Reply:
Either of them will have the same effect